### PR TITLE
add: check for issue link in PR body

### PR DESCRIPTION
**Pull request by gitbluf**
Add a step that checks if ISSUE link is provided.
Issue: https://github.com/schubergphilis/mcvs-pr-validation-action/issues/10

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -22,3 +22,15 @@ runs:
           exit 1
         fi
       shell: bash
+    - name: Check PR description for issue URL
+      shell: bash
```
Comment (sbp-bvanb) [inline action.yml:31]: This should be optional as there are also teams that use Jira instead of GitHub issues.

Comment (gitbluf) [reply action.yml:31]: Added optional input parameter.

File: .github/workflows/mcvs-pr-validation.yml [hunk 2]
```diff
@@ -17,3 +17,5 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.7
       - uses: ./
+        with:
+          gh_issue_check: true
```
Comment (sbp-bvanb) [inline .github/workflows/mcvs-pr-validation.yml:21]: Could you test this branch in the mcvs-determiner?

Comment (gitbluf) [reply .github/workflows/mcvs-pr-validation.yml:21]: It's tested => https://github.com/schubergphilis/mcvs-determiner/pull/142 (`on|off` as well as `break case`, you can look in actions|commit history)
But as I can see this repository uses the same code so it applies here as well.

**Review by sbp-bvanb — APPROVED** [review]

**Pull request merged**